src/higherOrders/withApolloClient/initApollo.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+// Set up using https://github.com/zeit/next.js/blob/canary/examples/with-apollo/lib/init-apollo.js
+import { ApolloClient, HttpLink, InMemoryCache } from 'apollo-boost'
+import fetch from 'isomorphic-unfetch'
+
+let apolloClient = null
+
+if (!process.browser) {
+  global.fetch = fetch
+}
+
+const create = (initialState: any) =>
+  new ApolloClient({
+    connectToDevTools: process.browser,
+    ssrMode: !process.browser,
+    link: new HttpLink({
+      uri: 'https://api.graph.cool/simple/v1/cixmkt2ul01q00122mksg82pn',
```

Comment:
Forgot I need to update that URL ☝️ will do that tomorrow using an env variable.